.github/workflows/frontend.yaml

```diff
@@ -0,0 +1,17 @@
+name: Frontend linting
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+    prettier:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - name: Install dependencies
+              run: npm install
```

Comment:
What package.json will this install? Will the next npx command run that version too?

Ideally we want it to be the one in app/frontend/package.json, so that it install and runs the same version devs use locally. Otherwise if npx uses the latest available version, it may be ahead and propose different changes.

Reply:
In the past I've added a prettier script in package.json, installed it and ran that command with npm run prettier. Not sure if there's a more 'Github workflow compatible' way of doing it.

Reply:
Ah good point. Let me tinker.

Reply:
I changed it to do everything inside "app/frontend" so it picks up the config from there. Also added a prettierignore there to ignore the JSON files, don't want to autoformat package-lock.json.